src/Statistics/include/Statistics/BinomialConfidenceInterval.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <cstdint>
+
+namespace orbit_statistics {
+
+struct BinomialConfidenceInterval {
```

Comment:
As per [google style guide](https://google.github.io/styleguide/cppguide.html#Class_Comments), please add a brief class level comments to the classes in this file.